### DISCUSSION TITLE
那个总存储和上传单个图片的大小在哪设置的啊？

### DISCUSSION THREAD

**Discussion by MrZhengI** [lsky-org/lsky-pro#404] [问答]
![image](https://user-images.githubusercontent.com/50481340/166058074-d65f2635-7ce5-4589-9e3b-8ee2a0872726.png)
![image](https://user-images.githubusercontent.com/50481340/166058137-f4fb5e5e-4d5c-4124-a62c-688af13a3136.png)

**Comment by MrZhengI** [comment]
总存储那个解决了，是在用户设置里面设置的，但是上传单个图片的大小和上传队列的该怎么设置呢，因为没接触过php，用的别人的docker镜像，修改了
![image](https://user-images.githubusercontent.com/50481340/166089773-92256955-0a35-40f7-a998-b18c1350222b.png)
这个，然后重启镜像，还是不行呢

**Reply by penndu** [reply]
> 总存储那个解决了，是在用户设置里面设置的，但是上传单个图片的大小和上传队列的该怎么设置呢，因为没接触过php，用的别人的docker镜像，修改了 ![image](https://user-images.githubusercontent.com/50481340/166089773-92256955-0a35-40f7-a998-b18c1350222b.png) 这个，然后重启镜像，还是不行呢

![image](https://user-images.githubusercontent.com/46226528/166095341-d5e7a66f-6485-4008-ba8f-ee8c1ee1d869.png)
![image](https://user-images.githubusercontent.com/46226528/166095347-e20c13fe-3dfb-4001-8c6b-7c1bd9b20654.png)

**Comment by 0xxb** [comment]
看文档
